### DISCUSSION TITLE
ubiquiti dream machine

### DISCUSSION THREAD

**Discussion by Th3Ic3Man** [Security-Onion-Solutions/securityonion#13870] [Unsupported Versions]
i am running a small home lab, my router is a ubiquiti dream machine. in the lastest early release now they support netflow and syslog.
i forgot, i am also a new user of security onion. i have to two questions:
1. will there be support for unifi routers in the future ? like pfsense
2. i manage to import netflow data to security onion, but from my understanding the suricata rules only apply to data from the sniffing interface ?

**Comment by TOoSmOotH** [comment]
Netflow should work exactly the same as OPN/PFsense. We are looking into further integration with the suricata components.
